ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml

```diff
@@ -438,6 +443,11 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: openshift-e2e-azure-csi
+- as: e2e-azure-csi-migration
+  interval: 12h
+  steps:
+    cluster_profile: azure4
+    workflow: openshift-e2e-azure-csi-migration
```

Comment:
We added 4.10 configs yesterday, can you add these to the the 4.10 file too? There's no automated forward propagation yet :(

Reply:
Added 4.10 jobs. I don't see 4.10 testgrids, I'll add them later.